src/store/modules/order/actions.ts

```diff
@@ -71,20 +71,6 @@ const actions: ActionTree<OrderState, RootState> = {
   async getOrderDetail({ commit, state }, { orderId }) {
     let resp;
 
-    const current = state.current as any
```

Comment:
This code is removed to fetch the data every time when we go to detail page, right?

Reply:
Yes, so as to avoid old item status issue due to slow Solr updation.